espnet2/bin/enh_inference.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+import logging
+import sys
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+from typing import Union
+
+import configargparse
```

Comment:
I changed to use espnet2.utils.config_argparse instead of it recently. The usage is almost same, so you can just replace it, please.

Reply:
In [espnet2/bin/enh_inference.py#L120-L121](https://github.com/espnet/espnet/blob/117fdc58404224ff32183af9202ccfeef2fdb9e7/espnet2/bin/enh_inference.py#L120-L121), we are using `configargparse.YAMLConfigFileParser` and `onfigargparse.ArgumentDefaultsHelpFormatter`. Should we change the arguments?

Reply:
Yes, you can remove configargparse.YAMLConfigFileParser and use argparse.ArgumentDefaultsHelpFormatter.

Reply:
I see the similar changes [here](https://github.com/espnet/espnet/commit/3564b086c545c3eedeff15e4aad8af3a82eeecb6#diff-b9d19c84a90511d45d133fe8f2b37ce0L220-L224). I'll just follow it by removing `config_file_parser_class=configargparse.YAMLConfigFileParser,` and replacing `formatter_class=configargparse.ArgumentDefaultsHelpFormatter,` with `formatter_class=argparse.ArgumentDefaultsHelpFormatter,`.

Reply:
Done.